commands.sh

```diff
@@ -0,0 +1,18 @@
+function Invoke-MB() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Tips depend on target shell and yours is unknown. Add a shebang.](https://app.codacy.com/manual/jrtapsell/MetabaseAutoCovid/pullRequest?prid=5378392)